src/CoreWCF.Http/tests/ClientContract/IEchoService.cs

```diff
@@ -30,5 +30,8 @@ public interface IEchoService
             ReplyAction = Constants.OPERATION_BASE + "EchoStreamAsyncResponse")]
         Stream EchoStreamAsync(Stream echo);
 
+        [OperationContract(Name = "EchoStringToFail", Action = Constants.OPERATION_BASE + "EchoStringToFail",
+     ReplyAction = Constants.OPERATION_BASE + "EchoStringResponse")]
+        string EchoToFail(string echo);
```

Comment:
ReplyAction should be EchoStringToFailResponse. This works on HTTP, but would likely break on NetTcp because it's duplex.